.github/actions/build_pandas/action.yml

```diff
@@ -25,8 +25,8 @@ runs:
     - name: Build Pandas
       run: |
         if [[ ${{ inputs.editable }} == "true" ]]; then
-          pip install -e . --no-build-isolation -v
+          pip install -e . --no-build-isolation -v  --config-settings=setup-args="-Db_sanitize=address,undefined"
```

Comment:
Probably don't want to hard code this - is there a way with GHA to only do this for certain action invocations @lithomas1 ?

Reply:
You can define an `input` above in the workflow and pass a variable from the job when you want to enable these flags

Reply:
I can push to your branch if you need any help with this, but it should be as Matt stated.

Reply:
Feel free to push